core/src/test/resources/org/bitcoinj/script/script_invalid.json

```diff
@@ -151,8 +151,8 @@
 ["2 2 LSHIFT", "8 EQUAL", "P2SH,STRICTENC", "disabled"],
 ["2 1 RSHIFT", "1 EQUAL", "P2SH,STRICTENC", "disabled"],
 
-["1","NOP1 NOP2 NOP3 NOP4 NOP5 NOP6 NOP7 NOP8 NOP9 NOP10 2 EQUAL", "P2SH,STRICTENC"],
```

Comment:
Before you merge the files in this PR, it would be better to do it in a separate PR and merge it before this one. I can take a look at the merging if you prefer to stay with working on the actual CSV stuff. I currently can't answer your question, have to dive deeper into it first.